src/screens/Login.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import * as Styles from '../styles/LoginStyles';
+import Container from '@material-ui/core/Container';
+
+class Login extends React.Component {
+  render() {
+    return (
+      <Container component="main" maxWidth="xs">
+        <Styles.MainDiv>
+          <form noValidate>
+            <Styles.FieldDiv>
+              <Styles.Label>Email</Styles.Label>
+              <Styles.Field
+                id="email"
```

Comment:
My comment earlier ab this line might have been a little confusing, but this is what I meant ab making the id descriptive! I just added a hotfix for that in my cleanup commit, but heads up for future